### PR TITLE
Cherry pick GetVehicleModelForCountry fix to release

### PR DESCRIPTION
**Pull request by tatiana-kondakova**
Cherry pick GetVehicleModelForCountry fix to release

### CODE REVIEW

**Comment by tatiana-kondakova** [comment]
DUDE JTTP

File: generator/routing_index_generator.cpp [hunk 1]
```diff
@@ -45,10 +47,13 @@ namespace
 class VehicleMaskBuilder final
 {
 public:
-  explicit VehicleMaskBuilder(string const & country)
-    : m_pedestrianModel(PedestrianModelFactory().GetVehicleModelForCountry(country))
-    , m_bicycleModel(BicycleModelFactory().GetVehicleModelForCountry(country))
-    , m_carModel(CarModelFactory().GetVehicleModelForCountry(country))
+  VehicleMaskBuilder(string const & country,
+                              CountryParentNameGetterFn const & countryParentNameGetterFn)
```
Comment (bykoianko) [inline generator/routing_index_generator.cpp]: clang-format pls

File: generator/generator_tool/generator_tool.pro [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ROOT_DIR = ../..
 
-DEPENDENCIES = generator routing traffic routing_common search storage indexer editor platform geometry \
+DEPENDENCIES = generator routing traffic routing_common search storage indexer editor mwm_diff platform geometry \
```
Comment (bykoianko) [inline generator/generator_tool/generator_tool.pro]: bsdiff?

File: generator/routing_index_generator.cpp (hunk 1 above)
Comment (tatiana-kondakova) [reply generator/routing_index_generator.cpp]: done

File: generator/generator_tool/generator_tool.pro (hunk 2 above)
Comment (tatiana-kondakova) [reply generator/generator_tool/generator_tool.pro]: в мастере нету

Comment (bykoianko) [reply generator/generator_tool/generator_tool.pro]: Ок

**Pull request merged**